src/plugins/intel_npu/src/compiler/src/driver_compiler_adapter.cpp

```diff
@@ -155,26 +119,27 @@ LevelZeroCompilerAdapter::LevelZeroCompilerAdapter() : _logger("LevelZeroCompile
         }
     }
 #endif
+
     if (ZE_GRAPH_EXT_VERSION_1_3 == targetVersion) {
         _logger.info("Using ZE_GRAPH_EXT_VERSION_1_3");
         apiAdapter =
-            std::make_shared<LevelZeroCompilerInDriver<ze_graph_dditable_ext_1_3_t>>(graphExtName, _driverHandle);
+            std::make_shared<LevelZeroCompilerInDriver<ze_graph_dditable_ext_1_3_t>>(graphExtName, _driverHandle, _deviceHandle, _context);
```

Comment:
Do we still need to get ext version here, backend already do that in https://github.com/KianYongGanIntel/openvino/blob/kgan/L0_debug/src/plugins/intel_npu/src/backend/src/zero_init.cpp#L43 and https://github.com/KianYongGanIntel/openvino/blob/kgan/L0_debug/src/plugins/intel_npu/src/backend/src/zero_init.cpp#L175